kubeflow-pipelines-visualization-server.advisories.yaml

```diff
@@ -162,17 +162,9 @@ advisories:
       - GHSA-5wvp-7f3h-6wmm
     events:
       - timestamp: 2024-01-19T19:23:18Z
-        type: detection
+        type: fixed
         data:
-          type: scan/v1
-          data:
-            subpackageName: kubeflow-pipelines-visualization-server
-            componentID: 1c7c197c61facfb2
-            componentName: pyarrow
-            componentVersion: 10.0.1
-            componentType: python
-            componentLocation: /usr/lib/python3.10/site-packages/pyarrow-10.0.1.dist-info/METADATA, /usr/lib/python3.10/site-packages/pyarrow-10.0.1.dist-info/RECORD, /usr/lib/python3.10/site-packages/pyarrow-10.0.1.dist-info/top_level.txt
-            scanner: grype
```

Comment:
I changed the `type` into `false-positive-determination` but CI complains again. Should we remove that existing data in this case?

Reply:
Kindly ping @luhring 😊

Reply:
It looks like you're still modifying existing data. The correct solution here is to append a new event for the false positive (if it's a false positive). You should be able to do that with `wolfictl adv update` but let me know if that doesn't work for you for some reason!

Reply:
Thank you, just issued that command and chose `inline-mitigations-exist`. But I'm not sure I chose the right one. Is it correct? Does it mean "we fixed it"?

Reply:
> Are we deleting the vulnerable code in our APK build pipeline now? Or what exactly is happening such that the vulnerability is gone?

Good questions! But I'm not sure if I clearly explain this since it would require some expert knowledge on the [pyarrow-hotfix](https://github.com/pitrou/pyarrow-hotfix) project. They basically [install](https://github.com/pitrou/pyarrow-hotfix/blob/69b94597e6f96707b7a7782468fac0920b1ad74a/src/pyarrow_hotfix/__init__.py#L31) some stuff to mitigate that CVE AFAICSC.

I know this wasn't a direct answer to your questions.

But found some explanation here: https://github.com/pitrou/pyarrow-hotfix/blob/69b94597e6f96707b7a7782468fac0920b1ad74a/src/pyarrow_hotfix/__init__.py#L7-L22

Reply:
Got it! Thanks for the context. I made a small suggestion — if that looks good to you, I think we can commit that, re-run CI, and merge this!

Reply:
-> https://github.com/wolfi-dev/advisories/pull/881#discussion_r1465510944

Reply:
Thank you! Applied the suggestion.